app/assets/stylesheets/base/_buttons.scss

```
@@ -20,7 +24,9 @@
 
   &--secondary {
     border: 1px solid $primary-dark;
+    background-color: $white-100;
     color: $black-100;
+    font-size: 0.875rem;
```

```suggestion
    font-size: rem(14px);
```
Don't you have `rem` mixin in the project? 🤔

--- 

I found out that we have it https://github.com/aetheryna/google-scraper-ruby/blob/feature%2Fb003-csv-upload/app/assets/stylesheets/functions/_sizing.scss 😛

---

Can you apply `rem` to other which is in the `px` unit?
But for some small amount we can omit that such as `1px `

---

Fixed, seems like I missed this one 🥲 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/35/commits/f347b499d2460bbd6f358239317f5953a1edd419